janus_core/helpers/mlip_calculators.py

```diff
@@ -224,6 +224,26 @@ def choose_calculator(
         kwargs.setdefault("sevennet_config", None)
         calculator = SevenNetCalculator(model=model_path, device=device, **kwargs)
 
+    elif arch == "orb":
+        from orb_models import __version__
+        from orb_models.forcefield.calculator import ORBCalculator
+        from orb_models.forcefield.graph_regressor import GraphRegressor
+        import orb_models.forcefield.pretrained as orb_ff
+        from  orb_models.forcefield.pretrained import orb_v2_mptraj_only as orb_mptraj_only_v2
+
+        model = getattr(orb_ff, model_path.sub("-", "_"), None)()
+        if model is None:
+                    raise ValueError(
+                        "Please specify `model_path`, as there is no "
+                        f"default model for {arch}"
```

Comment:
Is there no obvious default that makes sense, like we have for say mace_mp?

Reply:
Ah  there is `model = orb_ff.orb_v2()` below, but when should that be reached? If you don't pass anything, just above it raises `AttributeError: 'NoneType' object has no attribute 'sub'`